src/components/molecules/MapPartygoerOverlay/MapPartygoerOverlay.tsx

```diff
@@ -1,37 +1,34 @@
 import React, { useMemo } from "react";
 
 import { WithId } from "utils/id";
-import { User } from "types/User";
+import { Experience, User } from "types/User";
 
 import UserProfilePicture from "components/molecules/UserProfilePicture";
 
 interface MapPartygoerOverlayProps {
   partygoer: WithId<User>;
-
-  venueId: string;
-  myUserUid: string;
+  position: Experience;
```

Comment:
`Experience` contains far more data than what we actually want/need here. We only need a `row` and a `column`. Might even be worth just having 2 different props that are `number` for this, rather than passing an object with extra bits we don't need (which will also simplify memoisation checks)